### DISCUSSION TITLE
Can I use --chains option  in analyze command?

### DISCUSSION THREAD

**Discussion by Angieiscute** [milaboratory/mixcr#1599] [Q&A] [answered]
such as,

```
mixcr analyze generic-amplicon \
    --species hsa \
    --dna \
    --chains IGH \
    --floating-left-alignment-boundary \
    --floating-right-alignment-boundary J \
      input_R1.fastq.gz \
      input_R2.fastq.gz \
      result

```

I only want to export the chain I need. 
Thanks!

**Comment by mizraelson** [comment]
Hi,
What behavior do you expect? By default MiXCR identifies all chains which are present in the sample and splits the output by chains, when `analyze` command is used. There is no `--chains` parameter for analyze.

**Reply by Angieiscute** [reply]
We would like to only export the target chain we need, that's why we would like to use the `--chains` parameter for analyze.

**Reply by mizraelson** [reply, marked as answer]
I see, yes, you can use the following parameter with `mixcr analyze`: `-MexportClones.chains=IGH`.

**Reply by Angieiscute** [reply]
That works! Thank you!
